plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/HdfsInputFile.java

```diff
@@ -100,6 +100,13 @@ public int read()
             return delegate.read();
         }
 
+        @Override
```

Comment:
Should we be overriding `available` method too ? It looks like that would just return 0 instead of the actual bytes available from the delegate.

Reply:
related: https://github.com/trinodb/trino/pull/11322#issuecomment-1068933430

Reply:
@raunaqmorarka please follow up on this